docs/source/algorithms/EnggFitPeaks-v1.rst

```diff
@@ -36,14 +36,13 @@ the Mantid :ref:`ConvertUnits <algm-ConvertUnits>`. See also
 :ref:`EnggFitDIFCFromPeaks <algm-EnggFitDIFCFromPeaks>`.
 
 This algorithm currently fits (single) peaks of type
-:ref:`Back2BackExponential <func-Back2BackExponential>`. Other
+:ref:`Back2BackExponential <func-BackToBackExponential>`. Other
 alternatives might be added as optional in the future (for example the
 simpler :ref:`Gaussian <func-Gaussian>` or the more complex
-:ref:`Bk2BkExpConvPV <func-Bk2BkExpConvPV` or :ref:`IkedaCarpenterPV
-<func-IkedaCarpenterPV>`). To produce an initial guess for the peak
-function parameters this algorithm uses the :ref:`FindPeaks
-<algm-FindPeaks>` algorithm starting from the expected peaks list
-given in the inputs.
+:ref:`Bk2BkExpConvPV <func-NeutronBk2BkExpConvPVoigt>` or :ref:`IkedaCarpenterPV <func-IkedaCarpenterPV>`). To
```

Comment:
I just caught this by chance. We have a bit of confusion around these "back to back exponential" functions. Thanks for fixing these links!
Bk2BkExpConvPV is one of the undocumented functions: https://github.com/mantidproject/mantid/issues/16480.
NeutronBk2BkExpConvPVoigt has different parameters and can be more confusing than helpful. Maybe it's better to leave the text `Bk2BkExpConvPV` without a link?